### PR TITLE
TASK: Remove neos/fluid-adaptor require

### PR DESCRIPTION
**Pull request by sorenmalling**
As of neos/flow-development-collection#2151 the neos/fluid-adaptor is moved to a suggested package

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -30,7 +29,8 @@
         }
     },
     "suggest": {
-        "ext-pdo_sqlite": "For running functional tests out-of-the-box this is required"
+        "ext-pdo_sqlite": "For running functional tests out-of-the-box this is required",
+        "neos/fluid-adaptor": "For rendering HTML templates"
```
Comment (albe) [inline composer.json:33]:
```suggestion
        "neos/fluid-adaptor": "For rendering templates with TYPO3.Fluid"
```
Fluid can render more than HTML ;)

Comment (sorenmalling) [reply composer.json:33]: I will copy the same description to flow-development-collection :)

**Comment by sorenmalling** [comment]
@kdambekalns Can this be merged or is there any different routine when it's a distribution reopsitory?

**Review by kdambekalns — COMMENTED** [review]
Why is this even a requirement? It's a transitive dependency right now, so it should not be needed at all. It's not reequired in the base distribution, after all…

File: composer.json [hunk 2]
```diff
@@ -21,22 +21,7 @@
...
-        "neos/utility-arrays": "7.0.x-dev",
-        "neos/utility-files": "7.0.x-dev",
-        "neos/utility-mediatypes": "7.0.x-dev",
-        "neos/utility-objecthandling": "7.0.x-dev",
-        "neos/utility-opcodecache": "7.0.x-dev",
-        "neos/utility-pdo": "7.0.x-dev",
-        "neos/utility-schema": "7.0.x-dev",
-        "neos/utility-unicode": "7.0.x-dev",
-        "neos/http-factories": "7.0.x-dev"
+        "neos/behat": "dev-master"
```
Comment (albe) [inline composer.json:24]: See https://github.com/neos/flow-development-distribution/pull/56#issuecomment-740134735

**Review by albe — APPROVED** [review]

File: composer.json (hunk 2 above)
Comment (albe) [reply composer.json]: Will keep those for now

Comment (kdambekalns) [reply composer.json]: Nope, fine (see #56)

**Pull request merged**